service/client_state.cc

```diff
@@ -191,6 +192,31 @@ future<> service::client_state::has_access(const sstring& ks, auth::permission p
         }
     }
 
+    if (resource.kind() == auth::resource_kind::data && service::get_local_storage_service().cluster_supports_cdc()) {
+        const auto resource_view = auth::data_resource_view(resource);
+        if (resource_view.table()) {
+            static constexpr auto cdc_log_forbidden_permissions = auth::permission_set::of<
+                    auth::permission::CREATE, auth::permission::ALTER, auth::permission::DROP>();
```

Comment:
Maybe let's allow alter for now.

Reply:
Done, removed alter from this list.

I realized that CREATE has no meaning for tables, so I removed it as well.